components/fetchCalls.js

```diff
@@ -1,68 +1,87 @@
 const baseUrl = process.env.baseUrl;
-
+const form_id = process.env.form_id;
+const scopes = [
+  'posts',
+  'country_codes',
+  'media',
+  'forms',
+  'api',
+  'tags',
+  'savedsearches',
+  'sets',
+  'users',
+  'stats',
+  'layers',
+  'config',
+  'messages',
+  'notifications',
+  'webhooks',
+  'contacts',
+  'roles',
+  'permissions',
+  'csv'
+].join(" ");
 const fetchBearerToken = () => {
+  const tokenRequest = {
+    scope: scopes,
+    client_secret: process.env.client_secret,
+    client_id: process.env.client_id,
+    grant_type: "client_credentials"
+  };
   return fetch(baseUrl + "/oauth/token", {
     method: "POST",
-    body: JSON.stringify({
-      scope: "*",
-      client_secret: process.env.client_secret,
-      client_id: process.env.client_id,
-      grant_type: "client_credentials" //replace it with client_credentials
-      
-    })
+    body: JSON.stringify(tokenRequest),
+    headers: {
+      Accept: "application/json",
+      "Content-Type": "application/json"
```

Comment:
inconsistent quotation marks here. 
Use quotation marks for Accept and Authorization as well to improve readability